storage/innobase/xtrabackup/doc/source/prerequisites/configuring.rst

```diff
@@ -0,0 +1,44 @@
+.. _configuring:
+
+Configuring XtraBackup
+======================
+
+Setting options configures the *Percona XtraBackup*. These options behave
+exactly like the standard |MySQL| program options: they can be specified either at
+the command-line or through a configuration file like `/etc/my.cnf`.
+
+Please see the :ref:`option and variable reference
+<xbk_option_reference>` for details on all of the configuration options.
+
+The *Percona XtraBackup* binary reads the ``[mysqld]`` and ``[xtrabackup]`` sections from any configuration files, in that order. This order allows the binary to read options from your existing |MySQL| installation, such as the `datadir` or certain *InnoDB* options. If you must override these options, specify the options that must be overridden in the ``[xtrabackup]`` section. This section takes precedence since it is read later,
```

Comment:
```suggestion
The *Percona XtraBackup* binary reads the ``[mysqld]`` and ``[xtrabackup]`` sections from any configuration files, in that order. This order allows the binary to read options from your existing |MySQL| installation, such as the `datadir` or certain *InnoDB* options. If you must override these options, specify the options that must be overridden in the ``[xtrabackup]`` section. This section takes precedence since it is read later.
```

Reply:
Slight rewritten, fixed period